ocdskingfisherarchive/collection.py

```diff
@@ -13,14 +13,21 @@ def __init__(self, config, database_id, source_id, data_version):
         self.source_id = source_id
         self.data_version = data_version
         # The following attributes are filled by functions for caching purposes
-        self.data_md5 = None
-        self.data_size = None
+        self._data_md5 = None
+        self._data_size = None
+        self._scrapy_log_file_name = None
+        self._scrapy_log_file = None
 
     def write_meta_data_file(self):
+        self._cache_scrapyd_log_file_info()
         data = {
             'database_id': self.database_id,
             'data_md5': self.get_md5_of_data_folder(),
             'data_size': self.get_size_of_data_folder(),
+            'scrapy_log_file_found': (True if self._scrapy_log_file else False),
```

Comment:
Never do `True if condition else False` – it's unnecessarily complicated:

```suggestion
            'scrapy_log_file_found': bool(self._scrapy_log_file),
```

Reply:
Done